pkg/api/serialization_proto_test.go

```diff
@@ -38,7 +38,7 @@ import (
 
 func init() {
 	codecsToTest = append(codecsToTest, func(version unversioned.GroupVersion, item runtime.Object) (runtime.Codec, error) {
-		s := protobuf.NewSerializer(api.Scheme, runtime.ObjectTyperToTyper(api.Scheme))
+		s := protobuf.NewSerializer(api.Scheme, runtime.ObjectTyperToTyper(api.Scheme), "application/arbitrary.content.type")
```

Comment:
Yeah, I'm digging into why the general serialization is not roundtripping,
hopefully have that fixed tomorrow

On Mon, Apr 4, 2016 at 4:05 PM, Wojciech Tyczynski <notifications@github.com
> wrote:

> In pkg/api/serialization_proto_test.go
> <https://github.com/kubernetes/kubernetes/pull/23806#discussion_r58438663>
> :
>
> > @@ -38,7 +38,7 @@ import (
> >
> >  func init() {
> >  	codecsToTest = append(codecsToTest, func(version unversioned.GroupVersion, item runtime.Object) (runtime.Codec, error) {
> > -		s := protobuf.NewSerializer(api.Scheme, runtime.ObjectTyperToTyper(api.Scheme))
> > +		s := protobuf.NewSerializer(api.Scheme, runtime.ObjectTyperToTyper(api.Scheme), "application/arbitrary.content.type")
>
> OK - that makes sense. Thanks for explanation.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/23806/files/828d12e0f26fa80d52b22cc46d73f2f883b3b403#r58438663>
>